### ISSUE TITLE
Property expression query

### ISSUE THREAD

**Issue by carcaryn** [easybest/spring-data-mybatis#232]
Is it possibile to create an expression with path traversal? I put a simple example:

```
@Entity
class Emp {
     @Id
     Integer id;
     @ManyToOne
     @JoinColumn(name="id")
     Dept dept;
}

@Entity
class Dept {
     @Id
     Integer id;

     String name:
}
```
In emprepository i want a method with this sintax **List<Emp> findByDeptName(String name)**

**Comment by easybest** [comment]
the feature will be released in the next version 2.0.2, coming soon

**Comment by easybest** [comment]
It was implemented totally now.

https://github.com/easybest/spring-data-mybatis/blob/3859188d261251c068963177022b2e363c39f496/main/src/test/java/io/easybest/mybatis/repository/sample/UserRepository.java#L99
